sklearn/cluster/dbscan_.py

```diff
@@ -89,15 +95,15 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     """
     if not eps > 0.0:
         raise ValueError("eps must be positive.")
+    if random_state is not None:
```

Comment:
This should be a deprecation warning and should say that it will be removed in 0.18, I think.